core/string/ustring.h

```diff
@@ -429,6 +429,9 @@ class String {
 	String replace_first(const char *p_key, const char *p_with) const;
 	String replace(const String &p_key, const String &p_with) const;
 	String replace(const char *p_key, const char *p_with) const;
+	String replace_char(char32_t p_key, char32_t p_with) const;
+	String replace_chars(const String &p_keys, char32_t p_with) const;
+	String replace_chars(const char *p_keys, char32_t p_with) const;
```

Comment:
Can easily make a `replace_chars(const Vector<char32_t> &, char32_t)` version as well, it can use the same method, but leaving with this for now

Reply:
No need i think, there's little chance somebody even owns vector<char32_t> instead of String.